### PR TITLE
Vagrant local config

### PR DESCRIPTION
**Pull request by mkurek**
Added possibility to specify local Vagrant config (which will override default Vagrant settings).

### CODE REVIEW

**Comment by ar4s** [comment]
:ok:

File: .gitignore [hunk 1]
```diff
@@ -60,6 +60,8 @@ src/ralph/urls/*local.py
 
 # Vagrant
 vagrant/.vagrant
+vagrant/Vagrantfile.local
+vagrant/provision.sh.local
```
Comment (ghost) [inline .gitignore]: Maybe `vagrant/provision.local.sh`, so that editors would be aware of syntax.

**Comment by ghost** [comment]
:ok:

**Comment by szok** [comment]
:ok:

**Pull request merged**